drivers/soundwire/stream.c

```diff
@@ -1449,7 +1449,7 @@ struct sdw_dpn_prop *sdw_get_slave_dpn_prop(struct sdw_slave *slave,
 static void sdw_acquire_bus_lock(struct sdw_stream_runtime *stream)
 {
 	struct sdw_master_runtime *m_rt;
-	struct sdw_bus *bus = NULL;
+	struct sdw_bus *bus;
```

Comment:
what also makes such redundant initialisations easier to spot IMHO if respective variables are only declared in the scope where they're used, i.e. in the loop below in this case